src/qwed_new/phase4_runtime_probe.py

```
@@ -0,0 +1,56 @@
+
+
+def deserialize(blob):
+    return pickle.loads(blob)  # unsafe deserialization
+
+
+def load_config(text):
+    return yaml.load(text)  # yaml.load without Loader
+
+
+def get_user(conn, username):
+    query = f"SELECT * FROM users WHERE name = '{username}'"
```

Ruff (0.15.21)

[error] Possible SQL injection vector through string-based query construction

(S608)